kolibri/content/utils/check_schema_db.py

```diff
@@ -0,0 +1,54 @@
+import logging
+
+from sqlalchemy import inspect
+from sqlalchemy.ext.declarative.clsregistry import _ModuleMarker
+from sqlalchemy.orm import RelationshipProperty
+
+logger = logging.getLogger(__name__)
+
+# Modified from https://gist.github.com/miohtama/278fd4eeb9e5272d061c
+
+def db_matches_schema(Base, session):
```

Comment:
This might be slow, but it's steady, so will win the race.

Reply:
How often is this run? Every start?

Reply:
Looks like every start, whenever we import a database?

Reply:
Whenever we import a database. But not at application start.